apps/learner/src/lib/components/Starfield/Starfield.svelte

```diff
@@ -0,0 +1,269 @@
+<script lang="ts">
+  const stars = [
+    {
+      x: 677.273968659124,
```

Comment:
I don’t think the extra precision makes much difference. Should we consider limiting it to just 2~4 decimal places?